@here/harp-omv-datasource/lib/IGeometryProcessor.ts

```diff
@@ -30,17 +30,17 @@ export interface IPolygonGeometry {
     /**
      * The rings of this polygon.
      */
-    rings: IRing[];
+    readonly rings: IRing[];
 }
 
 /**
  * An interface to represent line geometries.
  */
 export interface ILineGeometry {
     /**
-     * The [[GeoCoordinates]] of the line.
+     * The positions of this [[IlineGeometry]] projected in world space.
```

Comment:
What world space? The one of the tiling scheme or the one of the map view? Please also add comments to the Vector3 above.
Maybe we should also consider to add a new type that adds additional information about the projection to Vector3 to make debugging and runtime checking easier (we would not even need to pass the source projection to the reproject method)

Reply:
good point, it's the world space of the tiling scheme. I will update the documentation.
In theory the processor (e.g. the decoders) that get these data have access to the TilingScheme. So I'm not sure we need more than a paragraph in the documentation mentioning that that the geo coordinates can be obtained by using `tilingScheme.projection`.

Reply:
For this change it's fine to just mention it.
I was just thinking that it might come in handy in the future if the world coordinates know what projection was used to produce them so the reprojectPoint would not need an additional sourceProjection parameter and we could also add some runtime checks in some places if the world coordinates are in the "wrong" world.